portfolio.css

```diff
@@ -0,0 +1,216 @@
+*{
+	box-sizing: border-box;
+	color: #fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
```

Comment:
not sure this is doing anything

Reply:
On smaller screen sizes without this the code tiles a second copy of the image below